docs/src/gui/qtvcp-vcp-panels.adoc

```diff
@@ -10,192 +10,198 @@
-- A combobox can be used to automatically select and connect to a pin/signal. +
-- You can add more LEDs from the drop down menu. +
-- The LED can be dtached from the main windows.
+- This panel has an *LED that can selected to watch HAL bit pins/signals*.
+- The LED's color can be adjusted from a drop down menu.
+- The text box and state can be output as speech if sound is selected.
+- A `combobox` can be used to automatically select and connect to a
+  pin/signal.
+- You can add more LEDs from the drop down menu.
+- The LED can be detached from the main windows.
 
-Here is how to load test_led from a HAL script. +
-The '-o' switch sets how many LEDs the panel starts with. +
-(If loading directly from a terminal omit the 'loadusr')
+Here is how to load `test_led` from a HAL script:
 
 [source,{hal}]
 ----
 loadusr qtvcp test_led
 loadusr qtvcp -o 4 test_led
 ----
 
-.test dial - Test Dial VCP
-image::images/qtvcp_test_led.png["Test LED",align="left"]
+The `-o` switch sets how many LEDs the panel starts with. +
+If loading directly from a terminal omit the 'loadusr'.
 
-=== test_panel
+.QtVCP `test_dial` Panel - Test LED VCP
+image::images/qtvcp_test_led.png["QtVCP test_dial Panel - Test LED VCP",align="center"]
 
-- collect of useful widgets for testing HAL component. Including speech
-  of LED state.
+[[sub:qtvcp:panels:test-panel]]
+=== `test_panel`
+
+*Collection of useful widgets for testing HAL component*, including
+speech of LED state.
 
 [source,{hal}]
 ----
 loadusr qtvcp test_panel
 ----
 
-.test panel - Test Panel VCP
-image::images/test_panel.png["Test Panel",align="left"]
+.QtVCP `test_panel` - HAL Component Testing Panel
+image::images/test_panel.png["QtVCP test_panel - HAL Component Testing Panel",align="center"]
 
-=== cam_align
+[[sub:qtvcp:panels:cam-align]]
+=== `cam_align`
 
-- a camera display widget for rotational alignment
+A *camera display widget for rotational alignment*.
 
-.QtVCP cam align- Camera based alignment panel
-image::images/qtvcp-cam-align.png["Qt Camera Alignment",align="left"]
+.QtVCP `cam_align` Panel - Camera Based Alignment Panel
+image::images/qtvcp-cam-align.png["QtVCP cam_align Panel - Camera Based Alignment Panel",align="center"]
 
-=== sim_panel
+[[sub:qtvcp:panels:sim-panel]]
+=== `sim_panel`
 
-- a small control panel to simulate MPG jogging controls etc. for
-  simulated configurations
+Small control panel to *simulate MPG jogging controls etc*. +
+For simulated configurations
 
 [source,{hal}]
 ----
 loadusr qtvcp sim_panel
 ----
 
-.QtVCP sim panel - Simulated controls for screen testing.
-image::images/qtvcp_sim_panel.png["Sim Panel",align="left"]
-
-[[cha:qtvcp-vismach-panels]]
+.QtVCP `sim_panel` - Simulated Controls Panel For Screen Testing.
+image::images/qtvcp_sim_panel.png["QtVCP sim_panel - Simulated Controls Panel For Screen Testing",align="center"]
 
-=== vismach_mill_xyz
+[[sub:qtvcp:panels:vismach]]
+=== `vismach_*` Panels
```

Comment:
I would shift this on level up and have "real subsections" inside
```suggestion
== `vismach_*` Panels
```

Reply:
Idea crossed my mind actually, but feared of over-structuring as this is really just a list of demo/sim panels...

But if you think so too, then it may not be that bad ;)